SignalServiceKit/src/Messages/Attachments/OWSAttachmentsProcessor.h

```diff
@@ -41,11 +41,11 @@ extern NSString *const kAttachmentDownloadAttachmentIDKey;
 
 - (void)fetchAttachmentsForMessage:(nullable TSMessage *)message
                     primaryStorage:(OWSPrimaryStorage *)primaryStorage
-                           success:(void (^)(TSAttachmentStream *attachmentStream))successHandler
```

Comment:
As I understand it, despite this "aggregate" completion handler, we're still saving each attachment and `touch`ing it's message with each individual downloaded attachment, meaning we'll still be able to update the UI incrementally, rather than having to wait for the entire album to be downloaded. :+1:

Reply:
That's the idea.  We can revisit as we implement the view changes.